moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -423,7 +432,7 @@ KinematicState Servo::getNextJointState(const ServoInput& command)
     joint_position_delta *= collision_velocity_scale_;
 
     // Compute the next joint positions based on the joint position deltas
-    target_joint_positions = current_joint_positions + joint_position_delta;
+    target_joint_positions = initial_joint_positions + joint_position_delta;
```

Comment:
Maybe i missed it but i don't see that `current_joint_positions` was modified at all between L410 and here. So why not use `current_joint_positions` and don't bother creating `initial_joint_positions`?